pkg/tar/tar_test.go

```diff
@@ -229,6 +229,7 @@ func TestCreateTar(t *testing.T) {
 		{"dir01/test2.git", modificationDate, 0660, "Test2 file content", false, ""},
 		{"dir01/dir03/test3.txt", modificationDate, 0444, "Test3 file content", false, ""},
 		{"dir01/.git/hello.txt", modificationDate, 0600, "Ignore file content", true, ""},
+		{"dir01/dir03/tëst3.txt", modificationDate, 0444, "Test utf-8 file header content", false, ""},
```

Comment:
Golang's tar library dynamically sets the format for each file in the tar archive. This tests that the tar streaming works with UTF-8 file names.

See https://golang.org/pkg/archive/tar/#Format